corehq/toggles.py

```diff
@@ -1828,10 +1828,19 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link='https://confluence.dimagi.com/display/ccinternal/Two-Stage+Mobile+Worker+Account+Creation',
 )
 
+LOCATION_REASSIGNMENT = StaticToggle(
```

Comment:
Not worth changing now, but it's helpful to put new toggles in the middle of the file, not the end, to reduce merge conflicts.